multiversion/src/main/scala/multiversion/resolvers/DependencyId.scala

```diff
@@ -9,7 +9,9 @@ final case class DependencyId(
     classifier: Option[String],
     dependencies: List[String],
     excludes: List[String]
-)
+) {
+  def repr: String = s"$organization:$name:$version"
+}
```

Comment:
Where are you using this new method?

Reply:
multiversion/src/main/scala/multiversion/diagnostics/EvictedDeclaredDependencyDiagnostic.scala I think

Reply:
It's used here: https://github.com/twitter/bazel-multiversion/blob/08ed7a12dfb8237e7c07d5fc83af7337f93b9460/multiversion/src/main/scala/multiversion/commands/ExportCommand.scala#L355
Comparing directly instances of `Dependency` is unreliable because they may have different excludes, etc.

Reply:
No you're right actually, it's not used at all !

Reply:
Good catch Angela 🚀